Automattic/wc-calypso-bridge
# Add payment OBW step styling

Fixes #83 

This PR adds styling for the payment step during the OBW.

### Screenshots
<img width="578" alt="screen shot 2018-10-26 at 3 37 29 pm" src="https://user-images.githubusercontent.com/10561050/47591504-7ab55b80-d935-11e8-95fd-20b009a90a63.png">

### Testing

1.  Visit `/wp-admin/admin.php?page=wc-setup&step=payment`
2.  Verify that the payment box styles match those of the hifi designs. p58i-7kf-p2
